es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,56 @@
+import createElement from '../lib/create-element';
+import noop from '../lib/noop'
```

Comment:
Here you can take advantage of the `index.js` file in the `lib` folder and import both of these on one line.

```JAVASCRIPT
import { createElement, noop } from '../lib'